src/core/sys/linux/sys/time.d

```diff
@@ -0,0 +1,72 @@
+/**
+ * D header file for POSIX.
+ *
+ * Copyright: Copyright Sociomantic Labs GmbH.
+ * License:   $(WEB www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Leandro Lucarella
+ */
+
+/*          Copyright Sociomantic Labs GmbH.
+ * Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ */
+module core.sys.linux.sys.time;
+
+private import core.sys.linux.config;
+public import core.sys.posix.sys.types;  // timeval
+
+version (linux)
+else static assert(false, "Unsupported platform");
+
+version( CRuntime_Glibc )
+else static assert(false, "Unsupported platform");
```

Comment:
In fact, that reveals an error in your code, the import of this file from `core.sys.posix.sys.time` is versioned on `linux` but not `CRuntime_Glibc`. On platforms that define only the former, druntime would not compile.